_translator-files/po/ko_KR/Running-a-Server.po

```diff
@@ -316,24 +292,18 @@ msgstr ""
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:101
 msgid "Note that Directories can only register up to 150 Servers. If you see a message that says you cannot register your Server because the Directory is full, you can try registering with a different Directory."
-msgstr ""
+msgstr "디렉터리는 최대 150개의 서버만 등록할 수 있습니다. 디렉터리가 가득 차서 서버를 등록할 수 없다는 메시지가 표시되면 다른 디렉터리에 등록해 볼 수 있습니다."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:105
-#, fuzzy, no-wrap
-#| msgid ""
-#| "<figure>\n"
-#| "\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Diagram of connections between Clients within a Jamulus Registered Server\">\n"
-#| "<figcaption>How Registered Servers work</figcaption>\n"
-#| "</figure>\n"
+#, no-wrap
 msgid ""
 "<figure>\n"
 "\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Diagram of connections between Clients within a Jamulus Registered Server\">\n"
 "</figure>\n"
 msgstr ""
 "<figure>\n"
-"\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Jamulus 등록 서버 내 클라이언트 간 연결 다이어그램\">\n"
-"<figcaption>등록된 서버의 작동 방법</figcaption>\n"
+"\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Jamulus 등록된 서버 내의 클라이언트 간 연결 다이어그램\">\n"
```

Comment:
If we have ko_KR screenshots this would be nice. I'd always prefer not to use the screenshots as includes.